### PR TITLE
[hi] Localizing `Reliability.md`

### LINKED ISSUE #1179: [hi] `Reliability` for `Hindi`
#### What this term is about? (briefly)
- Application reliability is the probability of a piece of software operating without failure.

#### Why this term needs to be added? (briefly) 
- Localizing this term in Hindi will help Hindi folks learn about this term in their native language.

Link here: https://glossary.cncf.io/reliability/

### PR DESCRIPTION
**Pull request by Omkar0114**
Signed-off-by: Omkar kulkarni <88308267+Omkar0114@users.noreply.github.com>
fixes #1179

### CODE REVIEW

File: content/hi/reliability.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: रिलियबिलिटी (Reliability)
+status: Completed
+category: गुण
+---
+
+क्लाउड नेटिव दृष्टिकोण से, विश्वसनीयता यह दर्शाती है कि सिस्टम विफलताओं के प्रति कितनी अच्छी तरह प्रतिक्रिया करता है। यदि हमारे पास एक वितरित प्रणाली है जो बुनियादी ढांचे में बदलाव के रूप में काम करती रहती है और व्यक्तिगत घटक विफल हो जाते हैं, तो यह विश्वसनीय है। दूसरी ओर, यदि यह आसानी से विफल हो जाता है और इसे चालू रखने के लिए ऑपरेटरों को मैन्युअल रूप से हस्तक्षेप करने की आवश्यकता होती है, तो यह अविश्वसनीय है। क्लाउड नेटिव एप्लिकेशन का लक्ष्य स्वाभाविक रूप से विश्वसनीय सिस्टम बनाना है।
```
Comment (sayantani11) [inline content/hi/reliability.md:7]: I think we should maintain consistency by using either `रिलियबिलिटी` or `विश्वसनीयता` everywhere.

**Review by sayantani11 — CHANGES_REQUESTED** [review]

File: content/hi/reliability.md (hunk 1 above)
Comment (Omkar0114) [reply content/hi/reliability.md:7]: Sure. I updated with `विश्वसनीयता` @sayantani11

**Review by sayantani11 — APPROVED** [review]
LGTM! 
Thank you

File: content/hi/reliability.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: विश्वसनीयता (Reliability)
+status: Completed
+category: गुण
+---
```
Comment (jayesh-srivastava) [inline content/hi/reliability.md:5]: We need to add the 'tags' column too. 
Ref: https://github.com/cncf/glossary/blob/main/content/en/reliability.md

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]

File: content/hi/reliability.md (hunk 2 above)
Comment (Omkar0114) [reply content/hi/reliability.md:6]: Updated :)

**Comment by Omkar0114** [comment]
Hi, @jayesh-srivastava 
I added tag

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM

**Comment by Omkar0114** [comment]
Thanks for merging 🥳

**Pull request merged**